www/include/configuration/configObject/traps/listTraps.php

```diff
@@ -86,6 +103,14 @@
 $lvl_access = ($centreon->user->access->page($p) == 1) ? 'w' : 'r';
 $tpl->assign('mode_access', $lvl_access);
 
+// Get vendors (manufacturers) for search dropdown
+$vendorResult = $pearDB->query("SELECT * FROM traps_vendor ORDER BY name, alias");
```

Comment:
if you need only the 'id' and 'name' column, don't pull every data from this table for better performances